composer.json

```diff
@@ -13,55 +13,55 @@
         "symfony/polyfill-php81": "*"
     },
     "require": {
-        "php": ">=8.1",
+        "php": ">=8.2",
```

Comment:
The README needs an update of the requirements as well to reflect this change.

Reply:
Good catch! I've just updated this. Thanks.